tests/main/interfaces-pulseaudio/task.yaml

```diff
@@ -1,98 +1,59 @@
 summary: Ensure that the pulseaudio interface works
 
-# Classic Ubuntu is sufficient to test the feature
-systems: [ubuntu-1*-*64, ubuntu-2*-*64]
+# Classic Ubuntu is sufficient to test the feature,
+# except Ubuntu 14.04 where systemd is not supported.
+systems: [ubuntu-16.04-*, ubuntu-18.04-*, ubuntu-19.10-*, ubuntu-20-*]
```

Comment:
I suppose we can drop 19.10 at this point.

Reply:
Let's drop it when it's dropped from systems-*